examples/llm_embedding.py

```diff
@@ -158,7 +158,7 @@ def __call__(self, sentences: List[str]) -> Tensor:
 stype_encoder_dict = {
     stype.categorical: EmbeddingEncoder(),
     stype.numerical: LinearEncoder(),
-    stype.text_embedded: LinearEmbeddingEncoder(),
+    stype.text_embedded: LinearMultiEmbeddingEncoder(),
```

Comment:
Can we rename all `LinearMultiEmbeddingEncoder` into `LinearEmbeddingEncoder` (given that we've deprecated the old `LinearEmbeddingEncoder`)?